static/js/components/ErrorMessage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import LoginButton from '../containers/LoginButton';
+import { Alert } from 'react-bootstrap';
+
+
+class ErrorMessage extends React.Component {
+  static propTypes = {
+    errorInfo:   React.PropTypes.object.isRequired,
+  };
+
+  state = {
+    alertVisible: true
+  };
+
+  render() {
+    if (!this.state.alertVisible) {
```

Comment:
In general we should refrain from using component state since it's more easily maintained in the redux store. `ErrorMessage` should take an additional `close` function in the `props` which should take care of making the error message go away.

Also, we should assume that `ErrorMessage` is not being shown if there is no error. For example here is a `render()` method:

    let errorMessage;
    if (hasError) {
      errorMessage = <ErrorMessage errorInfo={errorInfo} ... />;
    }

    <Page>
      {errorMessage}
    </Page>

`ErrorMessage` is only shown if there is something to show, so inside this class we don't need to check if it is visible or if the error message exists, for example.